packages/core-web/src/index.js

```diff
@@ -23,42 +23,54 @@ function scrollToUrlAnchorHeading() {
 }
 
 function detectAndApplyHeaderStyles() {
-  jQuery(':header').each((index, heading) => {
+  document.querySelectorAll('h1, h2, h3, h4, h5, h6').forEach((heading) => {
     if (heading.id) {
-      jQuery(heading).removeAttr('id'); // to avoid duplicated id problem
+      heading.removeAttribute('id');
     }
-  });
-
-  const headerSelector = jQuery('header[sticky]');
+  }); // to avoid duplicated id problem
+  const headerSelector = document.querySelectorAll('header[sticky]');
   if (headerSelector.length === 0) {
     return;
   }
   const [headerEl] = headerSelector;
 
-  let headerHeight = headerSelector.height();
+  function getHeaderHeight() {
```

Comment:
This is a little bit complicated because [.height() in jQuery does not include padding, border or margin](https://www.w3schools.com/jquery/css_height.asp#:~:text=The%20height()%20method%20sets,height%20of%20ALL%20matched%20elements.) but clientHeight includes [padding](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight). So the padding is calculated and deducted. 

imo this shouldn't affect much but did this to stick to the original implementation